scripts/build/install-tcmalloc.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -x
+set -ev
+
+DIR=$(dirname "$(readlink -f "$0" || stat -f "$0")")
+source "${DIR}/common-defaults.sh"
+
+if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+  # Get tcmalloc release
+  if [[ ! -e "${BASE}/gperftools-${TCMALLOC_VERSION}" ]]; then
+    cd "${BASE}"
+    wget "https://github.com/gperftools/gperftools/releases/download/gperftools-${TCMALLOC_VERSION}/gperftools-${TCMALLOC_VERSION}.tar.gz"
+    tar xfz "gperftools-${TCMALLOC_VERSION}.tar.gz"
+  fi
+  cd "${BASE}/gperftools-${TCMALLOC_VERSION}"
+  ./configure --disable-dependency-tracking --disable-cpu-profiler \
+    --disable-heap-checker --disable-debugalloc  --enable-minimal \
+    --prefix="${BASE}/tcmalloc-install"
+  make
+  make install
+  if [[ "${KEEP_BUILD}x" != "1x" ]]; then
```

Comment:
Why have this complexity? Just keep everything. If you want to have smaller dockerimages, use multiple FROM clauses.

Reply:
The script can be used outside of docker as well.
And it makes it currently easier for the docker build.